packages/elm/index.js

```diff
@@ -14,9 +14,10 @@ module.exports = elm
  * @param {string}                  [options.cwd]           Custom location for your elm files.
  * @return {Function}
  */
-function elm (options, isProduction) {
-  options = options || {}
-  isProduction = isProduction || process.env.NODE_ENV === 'production'
+function elm (options = {}, isProduction) {
```

Comment:
I think the idea was "let the user decide if he needs a development or production config, but use a sane default".

Suggestion: Let's open an issue for that if you think it is a problem.